src/Lib/ImageTransform.php

```diff
@@ -27,6 +29,28 @@ class ImageTransform
     private $Imagine;
 
     /**
+     * @var Table $table Instance of the table being used
+     */
+    protected $Table;
```

Comment:
True @jadb I've not tried the new cake3 standard agains the code base yet. I'll have to give it a try and see how badly I've done :wink: